File: src/main/java/arcana/common/particles/ModParticles.java
package arcana.common.particles;

import arcana.Arcana;
import net.minecraft.particles.ParticleType;
import net.minecraft.util.ResourceLocation;

public class ModParticles {
    public static ParticleType<MarkParticle.Data> markType = new ClientOnlyParticle.Type<>(true, new ResourceLocation(Arcana.id, "mark"));
}

File: src/main/kotlin/arcana/common/particles/ModParticles.kt
package arcana.common.particles

import arcana.utils.Util.arcLoc
import net.minecraft.particles.ParticleType

object ModParticles {
    @JvmField val MARK_TYPE: ParticleType<MarkParticle.Data> =
        ClientOnlyParticle.Type<MarkParticle.Data>(true).setRegistryName(arcLoc("mark")) as ParticleType<MarkParticle.Data>
    @JvmField val FIRE: ParticleType<LinearFireParticle.Data> =
        ClientOnlyParticle.Type<LinearFireParticle.Data>(false).setRegistryName(arcLoc("fire")) as ParticleType<LinearFireParticle.Data>
}